src/controllers/articlesController.js

```diff
@@ -1,97 +1,88 @@
 import { articlesService } from '../services/articlesService.js';
+import { getUserIdFromToken } from '../utils/token.js';
 
-export async function createArticle(req, res, next) {
-  try {
-    const { title, content } = req.body;
-    const newArticle = await articlesService.createArticleInDb(title, content);
+export async function createArticle(req, res) {
+  const { title, content } = req.body;
+  const userId = req.user.id;
 
-    res.status(201).json({
-      message: '게시글이 성공적으로 등록되었습니다.',
-      data: newArticle,
-    });
-  } catch (error) {
-    next(error);
-  }
+  const newArticle = await articlesService.createArticleInDb(title, content, userId);
+
+  res.status(201).json({
+    message: '게시글이 성공적으로 등록되었습니다.',
+    data: newArticle,
+  });
 }
 
-export async function getArticles(req, res, next) {
-  try {
-    const { sort, search } = req.query;
-    const { offset: _offset, limit: _limit } = req.paginationParams;
+export async function getArticles(req, res) {
+  const { sort, search } = req.query;
+  const { offset: _offset, limit: _limit } = req.paginationParams;
```

Comment:
이 친구는 따로 선언 없이 offset, limit 으로 바로 써도 될 것 같아요!

```suggestion
  const { offset, limit } = req.paginationParams;
```